docs/Getting Started/Debian/Debian Buster Root on ZFS.rst

```diff
@@ -51,7 +51,8 @@ Contributing
 
 #. Install the tools::
 
-    sudo apt install pip3
+    sudo apt install pip3 sensible-utils
```

Comment:
Why do we need `sensible-utils`? I prefer to have a minimal requirements to start.

Reply:
Because we call sensible-browser, which is a part of sensible-utils. I think this is a part of Debian by default though

Reply:
It should be installed, but even if it's not, it's not a big deal. The point is to show where the index.html is. I just use `sensible-browser` because it's a real command that will do the right thing. But it's not really necessary to install it.

Reply:
What do you think to move it into new `optional` section then?

Reply:
OK, I removed sensible-utils

Reply:
...Is this resolved. Should I close it?